puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-pipeline.groovy

```diff
@@ -13,6 +13,7 @@ pipeline {
             agent { label 'sshkey' }
             steps {
                 mash("foreman-mash-split.py", foreman_version)
+                mash("foreman-rails-mash-split.py", foreman_version)
```

Comment:
If we are mashing, we would need to also push them?

Reply:
well, yes, of course!

Reply:
it's now being pushed.